src/docs/references/stock/index.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Listing stock management
+slug: stock
+updated: 2021-09-13
+category: references
+ingress: Reference documentation for listing stock management.
+published: true
+---
+
+The listing stock management features of Flex allow providers to define
+the quantity of available stock for purchase. There are three key
+concepts related to stock management:
+
+- **stock** represents the quantity of available units, associated with
+  a listing. The units can be anything that makes sense for a given
+  listing or marketplace concept, as long as the quantity is integer
+  number. For instance - the number of physical items kept in inventory;
```

Comment:
I thought that negative stock was not allowed. "-99999" is an integer.

Reply:
In general yeah, but also https://github.com/sharetribe/sharetribe-custom/blob/stock-api-reference/doc/api/includes/marketplace/_stock.md. Maybe can link to that section or alternatively move the "negative stock section" to Flex docs and link to it from the API reference.